imports/api/NewGameUtils.js

```diff
@@ -56,38 +56,36 @@ export var NewGameUtils = {
 				return "西";
 			else //if (round > 12)
 				return "北";
-			break;
 		case Constants.GAME_TYPE.JAPANESE:
 			if (round <= 4)
 				return "東";
 			if (round > 4 && round <= 8)
 				return "南";
 			else //if (round > 8)
-				return "西";;
-			break;
+				return "西";
 		};
 	},
 
 	// Helper function to ensure all players are selected
 	allPlayersSelected() {
 		return (Session.get("current_east") != Constants.DEFAULT_EAST &&
-		        Session.get("current_south") != Constants.DEFAULT_SOUTH &&
-		        Session.get("current_west") != Constants.DEFAULT_WEST &&
-		        Session.get("current_north") != Constants.DEFAULT_NORTH);
+				Session.get("current_south") != Constants.DEFAULT_SOUTH &&
```

Comment:
Align these blocks with spaces.  For instance, this line should be two tabs followed by 8 spaces in order for the Session(...) code to align properly on all environments.